airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -43,7 +43,7 @@ class SqoopHook(BaseHook, LoggingMixin):
     :param verbose: Set sqoop to verbose.
     :type verbose: bool
     :param num_mappers: Number of map tasks to import in parallel.
-    :type num_mappers: str
```

Comment:
should have users pass it in as an `int` type because it will be converted to str later on, [here](https://github.com/apache/incubator-airflow/pull/2703/files#diff-8e77f042c2e060bbfd60828431a91e9bL131)